README.md

```diff
@@ -198,6 +198,62 @@ python  \
     --restore-from-checkpoint-path ${ESM2_650M_CKPT}
 ```
 
+##### Running with Pydantic configs
```

Comment:
Can you lead with a high level section describing the workflow? For example something along the lines of.

Our pydantic based trainer takes a config file as input. You have control over the training configuration, model configuration, etc in the config. To pre-populate a config with reasonable defaults for different standard model settings we also provide a `recipe` script. Together the standard training workflow is to first use the recipe script to create a pre-filled out config file for training, then edit the file with any specifics that you want to change for a particular training run, then train with that config.